src/ch02/trafficLight.test.ts

```diff
@@ -10,28 +14,53 @@ class Car {
   }
 }
 
-const CYCLE = [TrafficLight.RED, TrafficLight.YELLOW, TrafficLight.GREEN];
+// interface TrafficLight {
+//   isRed(): boolean;
+//   isYellow(): boolean;
+//   isGreen(): boolean;
+//   updateCar(): boolean;
+// }
```

Comment:
```suggestion
function doSomeThing() {
   interface TrafficLight {
     isRed(): boolean;
     isYellow(): boolean;
     isGreen(): boolean;
     updateCar(): boolean;
 }
}
```
의미없는 주석은 doSomeThing 함수에 담아서 실행 되지 않게 하는 것이 좋습니다.

Reply:
습관을 바꾸겠습니다. 해당 코드는 삭제했습니다.